mobile/src/bundles/common/helpers/transform-dto-values-to-multi-selector/transform-dto-values-to-multi-selector.ts

```diff
@@ -0,0 +1,17 @@
+import {
+    type AutocompleteMultiSelectorValue,
+    type HardSkillsResponseDto,
+} from '~/bundles/common/types/types';
+
+const transformDtoValuesToMultiSelector = (
+    payload: HardSkillsResponseDto | null,
+): AutocompleteMultiSelectorValue[] => {
+    return payload
+        ? payload.items.map((skills) => ({
+              label: skills.name,
+              value: skills.id,
```

Comment:
Do we have name and id in each Dto Value? If yes, then maybe it's better to refactor multiselector and use name, id?

Reply:
change it to name and id, it's better for code consistent anyway